templates/partials/answers/radio.html

```diff
@@ -1,10 +1,21 @@
 {% from "components/radios/_macro.njk" import onsRadios %}
 
-{{ onsRadios({
+{% set config = {
   "id": answer["id"],
   "name": answer["id"],
   "legend": question_title,
   "legendClasses": "u-vh",
   "radios": map_radio_config(form, answer),
   "error": error
-}) }}
+} %}
+
+{% if 'voluntary' in answer and answer["voluntary"] %}
```

Comment:
This can be simplified:
```suggestion
{% if answer.voluntary %}
```